### DISCUSSION TITLE
is the  version 0.16.0 ready for users?

### DISCUSSION THREAD

**Discussion by rochdiboudjehem** [znck/grammarly#245] [Q&A] [answered]
I was using VSCode 1.65 to avoid the Server error, then updated to 1.67. 
The Grammarly extension doesn't have any options like the "check option" and even with the keyboard shortcut it gives me `command 'grammarly.check' not found` until I roll back to older version.
One available option is connect to Text Mate if clicked shows me: 
`Now logging TM Grammar parsing. Press Stop when finished`.
Are we supposed to use this version 0.16.0 over the old one? or should we wait?

**Comment by LinqLover** [comment]
The last version of the extension that works for me is v0.14.0.

**Comment by znck** [comment]
There's no command to start. Will add it back. 

Extension auto starts for files defined in grammarly.patterns preferences option.

**Comment by rochdiboudjehem** [comment]
@znck how to connect to your premium account?

**Comment by znck** [comment]
Not implemented yet. SDK is missing one required method, will be done next week.

**Comment by rochdiboudjehem** [comment]
Finally, I could make it work thanks to @znck advice. (Grammarly extension 0.16.0, on VSCode 1.57.0)
I was astonished how fast and smooth it was! Wow!
Even though it is still missing premium features, I think it is wonderful. 
Many Thanks.

**Comment by ennioVisco** [comment]
Despite adding `**/*.tex` to patterns, I can't seem to make it start for LaTeX files. 
I'd recommend others in the same situation to stay at `v0.14` and wait that the implementation of the new APIs is complete.

P.s. Thanks a lot for your great work, can't wait to try the new experience 🥳

**Reply by dyokokawa6** [reply]
I agree with this opinion. I could not start with .tex files even in my environment, v0.18.1, and v0.16.
This project has great value. Thank you.

**Reply by ennioVisco** [reply]
> I agree with this opinion. I could not start with .tex files even in my environment, v0.18.1, and v0.16. This project has great value. Thank you.

@dyokokawa6 look at the screenshots down in this conversation. Basically, after v0.16 we only get inline marks, no grammar score and no sign-in. They will come back soon (but if you need them, you can just rollback to v0.14 and wait a bit to update).

**Reply by dyokokawa6** [reply]
Thanks for the answer, I updated to v0.22.0 and added **/*.tex to the setting and it worked fine. It is very comfortable.

**Comment by andresxvar** [comment]
> Despite adding `**/*.tex` to patterns, I can't seem to make it start for LaTeX files. 

tex works for me after I added `**/*.tex`, though not premium yet. Tested on release and preview

**Comment by ennioVisco** [comment]
> > Despite adding `**/*.tex` to patterns, I can't seem to make it start for LaTeX files.
> 
> tex works for me after I added `**/*.tex`, though not premium yet. Tested on release and preview

My bad, it *is* working, it's just that it doesn't show the user in the bottom bar anymore and I happened to be in a set of files with very few grammar errors 😅

**Comment by znck** [comment]
Added a quick guide to [README.md](https://github.com/znck/grammarly/tree/main/extension#readme)

<img width="1624" alt="image" src="https://user-images.githubusercontent.com/2596484/167444712-d4a179e4-9239-46ed-8ea8-129c2500cdf0.png">

**Comment by znck** [comment, marked as answer]
Version 0.16.0 is first version using Grammarly SDK. It is missing some features like connecting premium account, personal dictionary etc. PR #240 would add some of those features soon.
